### ISSUE TITLE
redundant link a11y failures

### ISSUE THREAD

**Issue by susanev** [chef/automate#3849]
## overview
across automate we are getting redundant link a11y failures, especially on the client runs and compliance nodes lists pages. this is prolly gonna need a collab between ux and ui teams to fix.

### What It Means
Adjacent links go to the same URL.
### Why It Matters
When adjacent links go to the same location (such as a linked product image and an adjacent linked product name that go to the same product page) this results in additional navigation and repetition for keyboard and screen reader users.
### How to Fix It
If possible, combine the redundant links into one link and remove any redundant text or alternative text (for example, if a product image and product name are in the same link, the image can usually be given alt="").

**Comment by susanev** [comment]
this has been deprioritized.
